exercises/practice/pig-latin/.meta/lib/example.dart

```diff
@@ -0,0 +1,34 @@
+class PigLatin {
+  String translate(String phrase) {
+    return phrase.split(' ').map(_translateWord).join(' ');
```

Comment:
The difficulty scale is from 1 to 10, but I wouldn't mind adding a test for case - Would that involve making a pull request on the problem-specification repo, or can individual tracks have additional tests that aren't in the spec?

Regardless, I'll edit the solution to work with different cases

Reply:
> can individual tracks have additional tests that aren't in the spec?

Definitely!. Individual tracks can add additional test cases. It's more about the language and showcasing it rather and helping in learning it, and problem specs are a starting point.

> The difficulty scale is from 1 to 10

Thanks!, I think more tests are always better. You want to help the learner as much as possible via tests. Making their implementation less error-prone.

Reply:
Awesome, thank you! I'll add those tests as well then - I should have time to make all of the changes tomorrow morning, if not later today

Reply:
Since pig-latin is about phonology and not orthography, pig latin is case insensitive.

I would prefer to change the problem description in problem-specifications rather than adding more test cases about this, as I don't think that they serve to make the exercise more interesting.